src/week2/findLargestValue/Test1.java

```diff
@@ -0,0 +1,19 @@
+package week2.findLargestValue;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Test1 implements HighestScore {
```

Comment:
please choose a meaningful name for the class that has some resemblance with interface